out/extension.js

```diff
@@ -291,6 +319,33 @@ function handleSplunkFile(context) {
     specConfigs[currentDocument] = specConfig
 }
 
+async function handleSpl2File(context, progressBar) {
+    if (spl2Client) {
+        // Client and server are already running, try refreshing for case of new document
+        const range = new vscode.Range(new vscode.Position(0, 0), new vscode.Position(0, 1));
+        const text = vscode.window.activeTextEditor.document.getText(range) || " ";
+        vscode.window.activeTextEditor.edit((editBuilder) => {
+            // To refresh language server make a harmless edit by replacing the first character
+            editBuilder.replace(range, text);
```

Comment:
There is a minor downside to this trick in that it will make the document appear to have unsaved changes, I personally think it's worth the tradeoff here in order to allow switching between documents on a single server as opposed to launching language servers per-document but let me know if you have other thoughts

Reply:
Just to confirm, it appears to have unsaved changes, but it doesn't, right?

Reply:
correct, you can see this in the first gif attached to the PR - the first doc `modules.json` was saved but then when we visited it we see the unsaved changes icon (although the doc was unchanged)
before:
![image](https://github.com/splunk/vscode-extension-splunk/assets/4960530/b98fdfaa-0b71-4490-a0f7-1389d0726975)

after:
![image](https://github.com/splunk/vscode-extension-splunk/assets/4960530/a547f001-804b-4378-b1e9-b7c8e861ddb6)